admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -2213,7 +2213,13 @@ window.seo = (function () {
   /** Hide or show the input product selector */
   function hideShowRedirectToProduct() {
     redirectTypeValue = redirectTypeElem.val();
-    if (redirectTypeValue === '404' || redirectTypeValue === '410') {
+    if (redirectTypeValue === '404'
+      || redirectTypeValue === '410'
+      || redirectTypeValue === 'default'
+      || redirectTypeValue === '200-displayed'
+      || redirectTypeValue === '404-displayed'
+      || redirectTypeValue === '410-displayed'
+    ) {
```

Comment:
Nitpick but you could make it a bit less verbose by doing

```
const redirectTypeValues = [
  '410',
  '404',
  'default',
  '200-displayed',
  '404-displayed',
  '410-displayed',
]

if (redirectTypeValues.contains(redirectTypeValue) {
```

Reply:
@NeOMakinG It's already in QA dude I don't want to ping everyone again. 😄